amundsen_application/static/js/components/TableDetail/OwnerEditor/index.tsx

```diff
@@ -213,23 +211,23 @@ export class OwnerEditor extends React.Component<OwnerEditorProps, OwnerEditorSt
       <div className='owner-editor-component'>
         { content }
         {
-          !this.state.readOnly &&
+          !this.state.readOnly && Object.keys(this.state.itemProps).length === 0 &&
```

Comment:
I'm not the biggest fan of `itemProps` being a map where the keys are the user email. Should we convert to an array somewhere upstream?